src/main/app/claims/interestRateClient.ts

```diff
@@ -0,0 +1,19 @@
+import { request } from 'client/request'
+import { claimApiBaseUrl } from 'claims/claimStoreClient'
+import { MomentFormatter } from 'utils/momentFormatter'
+import { InterestAmount } from 'claims/models/interestAmount'
+import { Moment } from 'moment'
+
+export class InterestRateClient {
+
+  static calculateInterestRate (amount: number, rate: number, startDate: Moment, endDate: Moment): Promise<InterestAmount> {
```

Comment:
I don't think that the types for request.get() match InterestAmount?
We usually serialise at this point.
REC